docs/wiki/Linux.md

```diff
@@ -0,0 +1,61 @@
+# Linux Guide
+
+## Installation
+
+Go to [Releases](https://github.com/Zagrios/bs-manager/releases) page and go to the latest release. Download the necessary build installer for your distro (see below).
+
+### Universal (flatpak)
+
+You are required to have `flatpak` installed your system. After installing that, download the `.flatpak` file in the releases and run the following command:
```

Comment:
General idiot-proofing the instructions:

You are required to have `flatpak` installed your system, if it is not installed then do so using your system's package manager ("sudo apt install flatpak" for Ubuntu, or "sudo pacman -S flatpak" for Arch). After installing, download the `.flatpak` file from the releases, CD to the download location, and run the following command:

Reply:
Is there really a case where someone wants to use the flatpak instead of the AUR package / deb, and they don't have it installed, and don't know how to install it? I could be wrong here, but it seems redundant to me.

Reply:
Yeah, I think other distro's should be listed here like [Fedora](https://developer.fedoraproject.org/deployment/flatpak/flatpak-install.html) and [NixOS](https://flatpak.org/setup/NixOS) which isn't Debian or AUR supported.